package.json

```diff
@@ -12,28 +12,35 @@
   },
   "scripts": {
     "build": "gulp",
-    "create-push-notifications-service-user": "node dist/tools/createPushNotificationsServiceUser.js",
-    "create_db": "sequelize db:create",
-    "create_missing_users": "node dist/tools/create_missing_users.js",
-    "delete_db": "sequelize db:drop",
-    "jenkins": "cross-env NODE_ENV=test yarn delete_db; cross-env NODE_ENV=test yarn create_db && cross-env NODE_ENV=test yarn migrate && cross-env NODE_ENV=test yarn test",
+    "create_push_notifications_service_user": "babel-node tools/createPushNotificationsServiceUser.js",
+    "jenkins": "cross-env NODE_ENV=test yarn mongo:delete_db && cross-env NODE_ENV=test yarn test",
     "lint": "esw -c .eslintrc *.js server config --color",
     "lint:fix": "yarn lint --fix",
     "lint:watch": "yarn lint --watch",
-    "migrate": "sequelize db:migrate",
-    "migrate:undo": "sequelize db:migrate:undo",
+    "mongo:create_missing_users": "babel-node tools/createMissingUsers.js",
+    "mongo:delete_db": "babel-node tools/deleteDb.js",
+    "mongo:fail_on_db_exists": "babel-node tools/failOnDbExists.js",
+    "mongo:migrate:up": "db-migrate --config migrate.config.json up",
+    "mongo:migrate:down": "db-migrate --config migrate.config.json down",
     "report-coverage": "coveralls < ./coverage/lcov.info",
     "serve": "node ./dist/index.js",
     "snyk-protect": "snyk protect",
+    "sql:create_missing_users": "babel-node sql/tools/create_missing_users.js",
+    "sql:create_db": "sequelize db:create",
+    "sql:delete_db": "sequelize db:drop",
+    "sql:migrate": "sequelize db:migrate",
+    "sql:migrate:undo": "sequelize db:migrate:undo",
+    "sql:update_uuid": "babel-node sql/tools/addUUID.js",
+    "sql:to_mongo": "babel-node sql/tools/toMongo.js",
     "start": "gulp serve",
     "start:debug": "cross-env DEBUG=amida-notification-microservice:* yarn start",
-    "test": "cross-env NODE_ENV=test mocha --opts .mocharc",
+    "test": "cross-env NODE_ENV=test yarn mongo:fail_on_db_exists && cross-env NODE_ENV=test mocha --opts .mocharc",
```

Comment:
This might get annoying since it fails if the test db already exists. Thoughts?

Reply:
`cross-env NODE_ENV=test yarn mongo:delete_db` will delete the db from `.env.test`

Are you suggesting we should also check with the user before deleting? Like a:

```
About to delete db "amida-notifications-test": are you sure? (y/N)
```

Reply:
Checking with the user like that would be nice, but I don't think it's required.

Reply:
Deleting before running tests, and checking with user when NODE_ENV != 'test' is now implemented